docs/pytutorial.rst

```diff
@@ -762,13 +764,27 @@ Without the explicit conversion you're likely to get an error such as::
 
   error: invalid conversion from ‘int’ to ‘tutorial::EnumTypeID’
 
-.. note:: Currently only the ``typedef`` is supported. There is no support
-          for adding the enumeration values for C and Fortran.
+A enum can also be fully defined to Python::
+
+    enums:
+    - decl: |
+          enum Color {
+            RED,
```

Comment:
Does this handle enums with explicit values? 
E.g.
```cpp
enum Color {
   RED = 10,
   GREEN = 20,
   BLUE = 30 };
```

Reply:
Yes it does. I think sidre presents another situation where the values are enums from another include file.  But I think that should be ok too.